CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## [2.0.4]
```

Comment:
Not a minor change.. this has the potential to break the layout of people using this module. 

Also replacing the H3 tags here with P doesn't seem to fit with the module in general, seems like a specific project change :-)